webapp/src/components/Navbar/Navbar.js

```diff
@@ -98,42 +136,98 @@ export default class Navbar extends React.PureComponent {
       return this.renderModalPage()
     }
 
-    return (
-      <div className="Navbar" role="navigation">
-        {this.renderLogoHeader()}
-        <div className="navbar-menu">
-          <Menu secondary stackable>
+    const menuItems = (
```

Comment:
_Maybe_ this should be `renderMenuItems`? to keep consistency

Reply:
✅ done (aa3d651)